### PR TITLE
create issue templates

### PR DESCRIPTION
**Pull request by Soapy7261**
the title?

### CODE REVIEW

**Comment by Soapy7261** [comment]
i don't know what github's yapping about still

**Comment by C0D3-M4513R** [comment]
You can add links to the new issue page. Please use that instead of that pointless Ketting issue template.

File: .github/ISSUE_TEMPLATE/issue-with-ketting--not-the-launcher-.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: 'Issue with ketting and not the launcher'
+description: 'Wrong repository!'
```
Comment (C0D3-M4513R) [inline .github/ISSUE_TEMPLATE/issue-with-ketting--not-the-launcher-.yml:2]: see comment

**Review by C0D3-M4513R — CHANGES_REQUESTED** [review]

**Comment by Soapy7261** [comment]
like that?

**Review by C0D3-M4513R — APPROVED** [review]

**Pull request merged**